pkg/destroy/aws/aws.go

```diff
@@ -1716,6 +1716,10 @@ func deleteRoute53(session *session.Session, arn arn.ARN, logger logrus.FieldLog
 
 	sharedZoneID, err := getSharedHostedZone(client, id, logger)
 	if err != nil {
+		if err.(awserr.Error).Code() == route53.ErrCodeNoSuchHostedZone {
+			logger.Warn("hosted zone in list of tagged resources but no longer exists, skipping route53 cleanup")
```

Comment:
can you drop this warning, this will be repeated again and again on user stdout.

none of the other notfound handling code adds logging,.

Reply:
/hold